### PR TITLE
Speedy miner testing

### PR DESCRIPTION
**Pull request by Thoralf-M**
# Description of change

Prevents this `TRY 1 SLOW [>720.000s] iota-sdk::mod pow::miner::get_miner_num_workers_score` from happening often

## Type of change

- Enhancement (a non-breaking change which adds functionality)

## How the change has been tested

Running the tests locally

## Change checklist

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by qrayven — DISMISSED** [review]

**Review by thibault-martinez — DISMISSED** [review]
Nice

File: sdk/tests/pow/miner.rs [hunk 1]
```diff
@@ -11,23 +11,48 @@ use iota_sdk::{
...
 fn get_miner_score() {
-    let miner = get_miner(4000);
+    let miner = get_miner(100);
     let mut bytes = rand_bytes(256);
 
     let nonce = miner(&bytes[0..248]).unwrap();
     bytes[248..].copy_from_slice(&nonce.to_le_bytes());
 
-    assert!(PowScorer::new().score(&bytes) >= 4000f64);
+    assert!(PowScorer::new().score(&bytes) >= 10f64);
```
Comment (thibault-martinez) [inline sdk/tests/pow/miner.rs]: You changed the score here but not in the builder?

Comment (Thoralf-M) [reply sdk/tests/pow/miner.rs]: Yes, thanks

Comment (thibault-martinez) [reply sdk/tests/pow/miner.rs]: Is 10 not so small it can be achieved by accident?

Comment (Thoralf-M) [reply sdk/tests/pow/miner.rs]: Everything can be achieved by accident with some probability :trollface: 
But yeah 100 should be fine as well, undid the change

**Review by thibault-martinez — APPROVED** [review]

**Review by kwek20 — APPROVED** [review]

**Pull request merged**